packages/harmony/src/components/button/FollowButton/types.ts

```diff
@@ -1,4 +1,17 @@
-import type { HTMLProps } from 'react'
+import type { ChangeEventHandler, ComponentPropsWithoutRef } from 'react'
+
+export type CheckboxProps = {
+  type: 'checkbox'
+  onChange?: ChangeEventHandler<HTMLInputElement>
+} & Omit<ComponentPropsWithoutRef<'input'>, 'chiildren' | 'size'>
+
+type InputProps =
```

Comment:
i might follow what selectable pill does, cause for that you dont need to unset onChange and checked